src/components/Drawer/Drawer.css

```diff
@@ -0,0 +1,41 @@
+.mdc-drawer{
+	display: flex;
+	flex-direction: column;
+	height: 99.5vh;
+	width: 6.25vw!important;
+	z-index: 1!important;
+	position: fixed!important;
+}
+.mdc-drawer__header{
+	height: 10vh;
+	border-bottom: 1px solid #e4e4e4;
+}
+.mdc-drawer__header-content{
+	width: 100%;
+	justify-content: center;
+	align-items: center!important;
+}
+.mdc-list{
+	padding: 0!important;
+}
+.mdc-list-item{
+	height: 10vh!important;
+	justify-content: center!important;
+	align-items: center!important;
+}
+.mdc-list-item:hover{
+	color: #3047FF!important;
+	cursor: pointer;
+}
+.mdc-list-item__text{
+	height: 24px;
+	width: 24px;
+}
+.material-icon{
+	color:#B8C2CC!important;
+}
+@media only screen and (orientation:portrait) {
+    .mdc-drawer{
+    	display: none!important;
+    }
+}
```

Comment:
always have a new line at then end of file. This can be configure within your text editor so you don't have to do it yourself everytime

Reply:
@geekayush many files have this issue fix this in your next PR

Reply:
@itaditya Fixed the extra lines. I'll keep this in mind.

Reply:
great